.github/workflows/check-commit-format.yml

```diff
@@ -20,18 +20,32 @@ jobs:
         env:
           BASE_SHA: ${{ github.event.pull_request.base.sha }}
           HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+          PR_AUTHOR: ${{ github.event.pull_request.user.login }}
+
         run: |
           IFS=$'\n' commit_messages=($(git log --pretty=format:"%s" $BASE_SHA...$HEAD_SHA))          
           
+          declare -a skip_pr_authors=(
+            "red-hat-konflux"
```

Comment:
add a comment that this is an RHTAP user please

Reply:
It's OK to add the comment for clarification purposes but RHTAP is now named Konflux:  
"Simultaneously, we will continue to work on the stack _formerly known as RHTAP_ by forming a new upstream under the name **Konflux**."

Reply:
@enriquebelarte thank you for the details
Anyway, I added a comment